services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -37,16 +37,88 @@ import { fetchPermits } from "@mds/common/redux/actionCreators/permitActionCreat
 import { createDropDownList } from "@mds/common/redux/utils/helpers";
 import { FORM } from "@mds/common/constants/forms";
 import RenderHiddenField from "../forms/RenderHiddenField";
+import AuthorizationSupportDocumentUpload from "./AuthorizationSupportDocumentUpload";
+import { IProjectSummaryDocument } from "@mds/common";
+import {
+  renderTextColumn,
+  renderDateColumn,
+} from "@mds/common/components/common/CoreTableCommonColumns";
+import DocumentTable from "@mds/common/components/documents/DocumentTable";
+import {
```

Comment:
I think this import isn't being used.  It can probably go.

Reply:
yes. Removed.